### DISCUSSION TITLE
How to set initial window position?

### DISCUSSION THREAD

**Discussion by kitaliev** [emilk/egui#1106] [Q&A] [answered]
Hi! I'm new in rust and using eframe framework. I see egui uses glium which uses winit and in winit you can set initial position with window builder's method with_position
And another not so related question
Is there a way to get user screen dimensions? some crate that i could use or something
Thx

**Comment by emilk** [comment, marked as answer]
Added in https://github.com/emilk/egui/commit/cbc53fbe2e933b68285d0c23e32a3f1f5777ab77
